### PR TITLE
Feature/beta invites

### PR DESCRIPTION
**Pull request by coderdan**
WIP

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -1,6 +1,22 @@
 require 'digest/sha1'
 
 class ApplicationController < ActionController::Base
+  before_action :beta_access
+
+  def beta_access
+    redirect_to go_invites_path unless has_beta_access?
```
Comment (maxious) [inline app/controllers/application_controller.rb]: What about letting people log in?

File: app/views/invites/show.html.haml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+
```
Comment (maxious) [inline app/views/invites/show.html.haml]: SHould this be in editorial? Who has access?

File: spec/models/invite_spec.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
```
Comment (maxious) [inline spec/models/invite_spec.rb]: Please make specs for creating, sending and using/failing to access via invites

**Comment by bjgordon** [comment]
Closing as this feature is parked for now

**Pull request closed without merging**